sidekiq-batch.gemspec

```diff
@@ -24,5 +24,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 2.1"
   spec.add_development_dependency "rake", "~> 13.0"
   spec.add_development_dependency "rspec", "~> 3.0"
-  spec.add_development_dependency "fakeredis", "~> 0.8.0"
+  spec.add_development_dependency "fakeredis", "~> 0.7.0"
```

Comment:
`fakeredis 0.8.0` cause the RSpec to fail because of https://github.com/redis/redis-rb/pull/918/files#diff-54fab56a3b12825ac3ed9243b67fd459R588



It's already fixed [here](https://github.com/guilleiguaran/fakeredis/pull/247) but version 0.9.0 is not yet released so I downgraded to have a passing test suite.